### PR TITLE
Add lofreq

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Description
Add `lofreq` biobase components:

* `call`
* `indelqual`

## Checklist before requesting a review

- [x] I have performed a self-review of my code

- [x] Conforms to the [Contributing guidelines](https://github.com/viash-hub/base/blob/main/CONTRIBUTING.md)

- [x] Proposed changes are described in the [CHANGELOG.md](https://github.com/viash-hub/base/blob/main/CHANGELOG.md)

- [x] I have tested my code with `viash ns test --parallel -q <name or namespace>`

```bash
viash ns test --parallel -q lofreq
The working directory for the namespace tests is /tmp/viash_ns_test3438401139745231693
           namespace        functionality             platform            test_name exit_code duration               result
              lofreq     lofreq_indelqual               docker                start                                        
              lofreq          lofreq_call               docker                start                                        
              lofreq          lofreq_call               docker     build_executable         0        2              SUCCESS
              lofreq          lofreq_call               docker              test.sh         0        1              SUCCESS
              lofreq     lofreq_indelqual               docker     build_executable         0        2              SUCCESS
              lofreq     lofreq_indelqual               docker              test.sh         0        2              SUCCESS
Not all configs built and tested successfully
  6 configs were disabled
  4/4 configs built and tested successfully
  ```

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Documentation
  - [ ] Bug fixes

### CODE REVIEW

File: src/lofreq/call/config.vsh.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    - name: Outputs
+      arguments:
+        - name: --output
+          alternatives: -o
+          type: file
+          description: |
+            Vcf output file [- = stdout].
+          required: true
+          direction: output
+          example: "output.vcf"
```
Comment (rcannood) [inline src/lofreq/call/config.vsh.yaml:51]: `--out` is renamed to `--output` but `--ref` is not renamed to `--reference`.

WDYT? Should `--output` be changed back into `--out` or `--ref` to `--reference`? ^^

File: src/lofreq/call/config.vsh.yaml [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    - name: Arguments
+      arguments:
+        - name: --region
+          alternatives: -r
+          type: string
+          description: |
+            Limit calls to this region (chrom:start-end) [null].
```
Comment (rcannood) [inline src/lofreq/call/config.vsh.yaml:58]: Make the default values a little more explicit:

```suggestion
            Limit calls to this region (chrom:start-end). Default: none.

File: src/lofreq/call/config.vsh.yaml [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+          type: file
+          description: |
+            List of positions (chr pos) or regions (BED) [null].
+          required: false
+          example: "regions.bed"
+        - name: --min_bq
+          alternatives: -q
+          type: integer
+          description: |
+            Skip any base with baseQ smaller than INT [6].
```
Comment (rcannood) [inline src/lofreq/call/config.vsh.yaml:72]: Make the default values a little more explicit:

```suggestion
            Skip any base with baseQ smaller than INT. Default: 6.

File: src/lofreq/indelqual/test_data/hg38_chr21.fa
Comment (rcannood) [inline src/lofreq/indelqual/test_data/hg38_chr21.fa:1]: How big is this file? Do you think we could find a smaller one?

**Review by rcannood — CHANGES_REQUESTED** [review]
Thanks for your PR! See comments :relaxed:

File: src/lofreq/indelqual/test_data/hg38_chr21.fa
Comment (KaiWaldrant) [reply src/lofreq/indelqual/test_data/hg38_chr21.fa:1]: about 50 MB :s

Comment (rcannood) [reply src/lofreq/indelqual/test_data/hg38_chr21.fa:1]: I'm not confortable yet adding a 50MB resource file to the repo :sweat_smile:

**Review by rcannood — APPROVED** [review]

**Pull request merged**